lib/active_model_serializers/railtie.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails/railtie'
+require 'action_controller'
+require 'action_controller/railtie'
```

Comment:
The AMS railtie *does* require that the action_controller/railtie has been loaded so that `Rails.configuration.action_controller` exists. But there isn't any normal that wouldn't have already been required. It should have been loaded by rails/all or even active_record/railtie (see the comment here https://github.com/rails/rails/blob/master/activerecord/lib/active_record/railtie.rb#L5).